sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -205,11 +205,11 @@ export interface MetricResultsResponseValuesItem {
 // @public
 export interface MetricsBatchOptionalParams extends coreClient.OperationOptions {
     aggregation?: string;
-    endtime?: string;
+    endTime?: string;
```

Comment:
Is `string` the right type for datetime properties?

Reply:
In the generated layer also, Autorest converts endtime to string based on the specification obtained from the swagger.

Reply:
Changed the datatype to `Date` and converted to string in the custom layer.